docs/docs-content/clusters/pcg/manage-pcg/remove-pcg.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: "Delete a PCG"
+title: "Delete a PCG"
+description: "Learn how to delete a Private Cloud Gateway (PCG) from Palette."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["pcg"]
+---
+
+You can delete a Private Cloud Gateway (PCG) from Palette. Deleting a PCG removes the PCG from Palette, along with the
+Palette services deployed on the PCG nodes. Deleting a PCG does not remove the underlying infrastructure resources, such
+as the virtual machines, or machines. Those resources must be removed manually from the infrastructure environment.
```

Comment:
```suggestion
as the virtual machines or machines. Those resources must be removed manually from the infrastructure environment.
```
In OpenStack, we also have to delete security groups, load balancer, network  ports, and release the floating IPs. :)